### DISCUSSION TITLE
MOD问题

### DISCUSSION THREAD

**Discussion by ghost** [Meloong-Git/PCL#2847] [提问] [answered]
一些之前可以用的mod现在没有了搜索不出来

**Comment by hejiehao** [comment]
比如？

**Reply by ghost** [reply]
连锁破坏
精妙背包一些常用的mod都没有了

**Reply by hejiehao** [reply]
https://www.mcmod.cn/class/3739.html
https://www.mcmod.cn/class/3004.html
这不有吗

**Reply by ghost** [reply]
这个上边一些前置mod下载不了

**Reply by MoYuan-CN** [reply]
![image](https://github.com/Hex-Dragon/PCL2/assets/109946426/edf56f72-08d6-4c87-96c9-153d6e1bcfe9)
这不是还有

**Reply by ghost** [reply]
这个链接打不开

**Reply by hejiehao** [reply]
![image](https://github.com/Hex-Dragon/PCL2/assets/98140773/79a03c1c-e1a5-4fb7-90cd-ce640ce86b13)
![image](https://github.com/Hex-Dragon/PCL2/assets/98140773/d7178be0-5402-4f79-8447-2335b03d27e5)
这里的每一个链接都可以点得开

**Reply by ghost** [reply]
我这里显示无法访问此页面

**Reply by MoYuan-CN** [reply]
截个图

**Reply by hejiehao** [reply]
截个图

**Reply by ghost** [reply]
![Uploading Desktop Screenshot 2023.11.04 - 20.45.01.45.png…]()

**Comment by ghost** [comment]
<img width="1280" alt="Desktop Screenshot 2023 11 04 - 20 45 01 45" src="https://github.com/Hex-Dragon/PCL2/assets/149858128/066ba635-51a1-48bb-87aa-4c9bd3f64104">

**Reply by hejiehao** [reply]
建议开梯子

**Reply by ghost** [reply]
好的

**Reply by MoYuan-CN** [reply]
你的图全部都没有传上来

**Reply by ghost** [reply]
有什么好用的加速器吗那、

**Reply by MoYuan-CN** [reply]
对于 Github 和中国大陆来讲，[Steam++（Watt Toolkit）](https://steamapp.net)是一个很好的选择

**Reply by ghost** [reply]
有什么可以加速mod网站的加速器吗

**Comment by MoYuan-CN** [comment]
如还有问题请继续提供信息

**Reply by ghost** [reply]
有什么可以加速mod网站的加速器吗

**Comment by WForst-Breeze** [comment, marked as answer]
如果无法加载搜索结果，请检查您的网络环境，使用加速器或 VPN；如果点击 “转到 CurseForge” 时无法访问，请在浏览器中安装 Adblock Plus 插件并在 AdblockPlus 中 设置-高级-我的过滤列表 中添加以下过滤项后再试。
```
apis.google.com/js/platform.js$domain=minecraft.curseforge.com|www.curseforge.com
fonts.googleapis.com/css?family=*$domain=minecraft.curseforge.com|www.curseforge.com
tags.crwdcntrl.net/*.js$domain=minecraft.curseforge.com|www.curseforge.com
www.google-analytics.com/*.js$domain=minecraft.curseforge.com|www.curseforge.com
```

若你需要相关加速器或 VPN，很抱歉我无法为你提供 :(
